src/main/java/bio/terra/cli/command/resources/CheckAccess.java

```diff
@@ -12,21 +13,17 @@
     name = "check-access",
     description = "Check if you have access to a referenced resource.")
 public class CheckAccess extends BaseCommand {
-  @CommandLine.Option(
-      names = "--name",
-      required = true,
-      description = "Name of the resource, scoped to the workspace.")
-  private String name;
+  @CommandLine.Mixin ResourceName resourceNameMixin;
 
-  @CommandLine.Mixin FormatOption formatOption;
+  @CommandLine.Mixin Format formatOption;
```

Comment:
I prefer the convention of `ResourceName resourceNameMixin` because I think it makes accessing the value slightly more readable (e.g. `cloningMixin.cloning` instead of `cloning.cloning`).